site/docs/release-notes/3-14.md

```diff
@@ -3,13 +3,83 @@
+
+    ??? info "Use chromium-based or Safari browser for UIs"
+         It is recommended to use a chromium-based browser such as Google Chrome, Microsoft Edge or Apple Safari for the Egeria Ecosystem UI. Some parts of the UI experience such as Dino currently experience problems with Firefox. See [odpi/egeria-react-ui#96 :material-dock-window:](https://github.com/odpi/egeria-react-ui/issues/96){ target=gh }.
+
+    ??? info "Ecosystem UI minor issues"
+
+        We are continuing to develop the Ecosystem UI & improve it's capability and usability. These are not regressions in this release. Known issues include:
+
+        * [odpi/egeria-react-ui#461](https://github.com/odpi/egeria-react-ui/issues/463){ target=gh } - Dino: cannot retrieve server audit log twice.
+        * [odpi/egeria-react-ui#465](https://github.com/odpi/egeria-react-ui/issues/463){ target=gh } - Dino: Error retrieving cohort information in Dino.
+        * [odpi/egeria-react-ui#464](https://github.com/odpi/egeria-react-ui/issues/464){ target=gh } - Buttons can appear off screen. Workaround is to enlarge window.
+        * [odpi/egeria-react-ui#465](https://github.com/odpi/egeria-react-ui/issues/465){ target=gh } - Search dialog still appears if no server selected.
+
+        See [odpi/egeria-react-ui](https://github.com/odpi/egeria-react-ui/issues){ target=gh } for all issues or to open a new one on this UI.
+
+    ??? info "Egeria UI issues"
+        See [odpi/egeria-ui](https://github.com/odpi/egeria-ui/issues) for all issues or to open new one for Egeria UI.
+
+    ??? info "Documentation Link errors"
+
+        We are aware of broken links in the documentation here, as well as broken links to the documentation contained within the Egeria code -
+        for example links to component information.
+
+        We are trying to fix these as we update the docs, but please raise any additional problems found in [egeria-docs](https://github.com/odpi/egeria-docs/issues){ target=gh }.
+        We would also be very grateful for any PRs that fix issues.
+
+??? functional "Helm Chart versions"
+
+    These charts may get updated more frequently than Egeria code releases, as we integrate other components and fix bugs. You can always check the latest with `helm repo update && helm search repo egeria `
+
+    Note that the Business UI remains at an older version, as the code is currently undergoing some refactoring.
+
+    To see known issues/upcoming work or to report issues see [egeria-charts](https://github.com/odpi/egeria-charts/issues){ target=gh }
+
+    ??? info "Tested environments"
+
+        The Egeria charts have been tested in:
+        
+        * Rancher Desktop v1.6.2
+        * Kubernetes v1.25
+
+    ??? info "Lab Chart for 3.14"
+
+         The Egeria lab chart 3.14.0 has been updated to include:
+
+         * Egeria 3.14
+         * Egeria Ecosystem UI 3.8.0
+         * Egeria Business UI 3.2.2
+         * Strimzi 0.32.0 with Kafka 3.2.3
```

Comment:
3.3.1 with the new update to the charts (applies to further entries in this file)

Reply:
good sport... updated.